addons/binding/org.openhab.binding.sleepiq/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SleepIQ Binding
+Bundle-SymbolicName: org.openhab.binding.sleepiq;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/sleepiq-api.jar
+Import-Package: 
+ com.eclipsesource.jaxrs.provider.gson,
```

Comment:
The main problem I see is that the upstream API library needs to be usable outside of an OSGi context. While it should be possible to register the GsonProvider as a service (it has an activator), I don't have a way to inject it into the API library without making the library dependent on OSGi.

I will spend some time researching on this, but in the short term I don't think it is avoidable. I am open to suggestions though. While I have some experience with OSGi, I am by no means an expert.